src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -25,6 +26,73 @@ UiModules
       $rootScope.toggleRefresh = () => {
         timefilter.refreshInterval.pause = !timefilter.refreshInterval.pause;
       };
+
+      // functions for the time navigation buttons
+
+      // travel forward in time based on the interval between from and to
+      $scope.foward = function () {
+        var time = getFromTo();
```

Comment:
These should all be `const` unless you will be overriding the reference, in which case they would be `let`.